### ISSUE TITLE
[Suggestion] Add a buy all command

### ISSUE THREAD

**Issue by DhalEynn** [cmooref17/Lethal-Company-TooManyEmotes#84]
Hello,

We found out that in our games, we quickly had enough funds to buy every emotes on the rotation.
However, it is very long to do that emote by emote, especially since we have a 10 emotes rotation.

Would it be possible for you to add a buy all available emotes command, which would let you buy the [number of emotes in rotation] emotes with one command ?

Thanks you in advance for your answer, and have a nice day !

**Comment by cmooref17** [comment]
It could be done, but I wouldn't add logic related to which emotes should be unlocked if you can only afford some of them. That decision would be up to the player. If I were to add this, I wouldn't allow the command to run at all unless you could afford them all. Does that sound fair?

**Comment by DhalEynn** [comment]
Yes, that was my idea too.
The command would be for those who can afford every emote displayed on the shift rotation, not to pick and select which emote you want.
Thanks for the answer !

**Comment by cmooref17** [comment]
Alright, what would you like the command to be? Would "emotes buyall" work?
I will likely require the emotes keyword to ensure there will be no compat issues.

**Comment by DhalEynn** [comment]
Yeah, emotes buyall is perfect (or maybe emotesbuyall to avoid spaces in the command ? Idk)
Make it the easiest for you while being clear in what the command do :)
